### DISCUSSION TITLE
Welcome to mmyolo Discussions!

### DISCUSSION THREAD

**Discussion by hhaAndroid** [open-mmlab/mmyolo#114] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nanxnan** [comment]
用官方给的yolov8的代码跑了钢板缺陷数据集map很高，但是用mmyolo的yolov8跑同样的数据集map很低，是怎么回事呀

**Comment by maydo82** [comment]
Hi, I am working on joint network including enhancement and detection task and trying to use mmyolo due to its flexibility and I guess it will be easier to modify backbone and corresponding sections for further studies. I wanted to try basic example on mmyolo and installed the dependencies from https://github.com/open-mmlab/mmyolo website. Then just tried to implement the example on https://mmyolo.readthedocs.io/en/latest/get_started/15_minutes_object_detection.html, however it doesn't work. Anyone who will help?

**Comment by Vinaygoudasp7** [comment]
Hi @hhaAndroid @maydo82 @MambaWong,

I'm using MMYOLO and have successfully trained a model that gives good results. Now, I want to run the model on CPU by converting it to OpenVINO. I followed the conversion pipeline: MMYOLO → ONNX → OpenVINO (using the OpenVINO CLI).

However, after conversion, the inference results are not as expected. Could you please suggest possible solutions or guide me on how to resolve this?

Thanks in advance!

**Comment by Vinaygoudasp7** [comment]
hi any one can you give solutions for this

**Comment by collinmccarthy** [comment]
I think this project has folded. I follow all the issues and I haven't seen a significant amount of activity from maintainers in a long time. Try mmdetection which has more community support, or ultralytics. If you really need yolo, probably try ultralytics first (they're quite responsive). Good luck.

**Comment by Vinaygoudasp7** [comment]
@collinmccarthy  ok thanks for your suggestions but i need to try mmyolo with openvino so is any help me ?

**Reply by collinmccarthy** [reply]
Not sure, I'm guessing you're on your own.. good luck 🤞

**Comment by Vinaygoudasp7** [comment]
@collinmccarthy ok any one can know why predictions are giving empty array

**Comment by Vinaygoudasp7** [comment]
hi @hhaAndroid @hanrui1sensetime @maydo82 @MambaWong can any one tell me the preprocess and post process of of mmyolo(yolox) after converted to rknn. i followed the converstion as below  mmyolo easydeploy to onnx and to rknn using airockchip rknn toolkit2

**Comment by Vinaygoudasp7** [comment]
hi any one know about this  @hhaAndroid @hanrui1sensetime @maydo82 @MambaWong @collinmccarthy

**Comment by surprise335** [comment]
<img width="3250" height="3402" alt="微信图片_20250723145538" src="https://github.com/user-attachments/assets/b7e21f48-0003-4bff-ac8b-fda92d179f96" />
 would like to ask what software was used to create this image. Thanks.

**Comment by dock3-sudo** [comment]
Hi. I'm new here.

My mission is to create object detection models for clients using DJI Enterprise Drones with AI algorithm capabilities. Since DJI explicitly recommend mmyolo in their development documentation I took a leap and started off setting up my env. At first I had issues with installation of mmcvrc2.0.4 but fixed that. Then when I started with a very small dataset just to see if my setup is correct and test the training process ( dry run ) I kept receiving errors on top of errors. Then the last error I got glued to was about Key 'img_path' so got stuck on that and yes, recently fixed that aswell. Did a dry run train on only 5 images and the training completed without any conspiracy 😂 So the main culprit for the Key 'img_path' error was with albumentations, the 'albu' code line in the config file. Currently I'm now training a small dataset of 203 images ( a bit excited that's why image count is less than 500 ) and training on CPU because my laptop don't have graphics card. So at this point I'm very happy with it at the moment. Hope that this project works out for any and all. I have to mention that to setup and run Ultralytics is far more less technical than the mmyolo project but ya, DJI strictly wants devs to use mmyolo for the development of AI algorithms on their products. Wishing all of you here good luck and blessings.
